QualityControl/lib/services/ccdb/CcdbService.js

```diff
@@ -126,18 +125,24 @@ export class CcdbService {
    *
    * Attributes of objects wished to be requested for each object can be passed through the fields parameter;
    * If attributes list is missing, a default minimal list will be used: PATH, CREATED, LAST_MODIFIED
-   * @example Equivalent of URL request: `/latest/qc/TPC/object.*`
+   * @example Equivalent of URL request: `/latest/qc/TPC/object.* /RunNumber=42`
    * @param {string} [prefix] - Prefix for which CCDB should search for objects
+   * @param {object} [filters] - Object metadata that will be used to construct a endpoint path.
    * @param {Array<string>} [fields] - List of fields that should be requested for each object
    * @returns {Promise.<Array<{PATH, CREATED, LAST_MODIFIED}>>} - results of objects query or error
-   * @rejects {Error}
    */
-  async getObjectsLatestVersionList(prefix = this._PREFIX, fields = []) {
-    const headers = {
-      accept: 'application/json',
-      'x-filter-fields': fields.length > 0 ? fields.join(',') : `${PATH},${CREATED},${LAST_MODIFIED}`,
-    };
-    const { objects } = await httpGetJson(this._hostname, this._port, `/latest/${prefix}.*`, { headers });
+  async getObjectsLatestVersionList(
+    prefix = this._PREFIX,
+    filters,
+    fields,
+  ) {
```

Comment:
Consider options object:

 ```suggestion
  async getObjectsLatestVersionList({ prefix = this._PREFIX, filters = {}, fields = [PATH, CREATED, LAST_MODIFIED] } = {}) {
```

Reply:
I'll make an options object, but I won't set fields to default [PATH, CREATED, LAST_MODIFIED] as that would mean that an empty array could reach this function.

Reply:
yes, filters and fields can be mixed in order by mistake